modules/cluster-logging-collector-log-forward-project.adoc

```diff
@@ -19,8 +19,8 @@ To configure forwarding application logs from a project, you must create a `Clus
 apiVersion: logging.openshift.io/v1
 kind: ClusterLogForwarder
 metadata:
-  name: instance <1>
-  namespace: openshift-logging <2>
+  name: <log_forwarder_name> <1>
```

Comment:
Please revert changes here.   we should use name: instance , namespace: openshift-logging  here